cypress/page/address-step.page.ts

```diff
@@ -3,7 +3,7 @@ class AddressPage {
     private processToCheckoutButton: string;
 
     constructor() {
-        this.processToCheckoutButton = ".cart_navigation > .button > span";
+        this.processToCheckoutButton = '.cart_navigation > button[name="processAddress"] > span';
```

Comment:
Why not just `button[name="processAddress"]`

Reply:
Done!